src/main/java/com/connexta/ingest/controllers/IngestController.java

```diff
@@ -35,7 +38,14 @@
 
   @Override
   public ResponseEntity<Void> ingest(
-      String acceptVersion, MultipartFile file, String correlationId, MultipartFile metacard) {
+      String acceptVersion,
+      @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) OffsetDateTime lastModified,
+      MultipartFile file,
+      String correlationId,
+      MultipartFile metacard) {
+    if (lastModified == null || lastModified.toString().isBlank()) {
+      throw new ServerWebInputException("Last-Modified is missing or blank");
```

Comment:
✏️ `LAST_MODIFIED`

Reply:
`LAST_MODIFIED + " is missing or blank"`

Reply:
ok... maybe this time I got it